### PR TITLE
Adding issue and PR templates

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
Adding bug report and add organisation issue templates. 
Adding PR template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/add_org.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      description: Crunchbase link of organisation goes here
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Select in which list organisation is to be added
+      options:
+        - "Adopters List"
+        - "Community List"
+        - "Sponsors List"
```
Comment (benjagm) [inline .github/ISSUE_TEMPLATE/add_org.yml:48]:
```suggestion
      options:
        - "Adopters"
        - "Sponsors"
        - "Other"
```

File: .github/ISSUE_TEMPLATE/add_org.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      description: Name of organisation goes here
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Organisation Logo
+      description: Logo of organisation goes here
+    validations:
+      required: true
+  - type: textarea
```
Comment (benjagm) [inline .github/ISSUE_TEMPLATE/add_org.yml:18]: I'll suggest to request just the basic information at this first stage of discussion. Will get the logo later.

File: .github/ISSUE_TEMPLATE/add_org.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      label: Organisation Twitter link
+      description: Twitter link of organisation goes here
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Organisation Crunchbase link
+      description: Crunchbase link of organisation goes here
+    validations:
+      required: false
```
Comment (benjagm) [inline .github/ISSUE_TEMPLATE/add_org.yml:41]: I'll suggest to request just the basic information at this first stage of discussion. Will get these details later.

**Review by benjagm — CHANGES_REQUESTED** [review]
Great work Dhairya. Just left some comment to address before merging.

File: .github/ISSUE_TEMPLATE/add_org.yml (hunk 2 above)
Comment (DhairyaMajmudar) [reply .github/ISSUE_TEMPLATE/add_org.yml:18]: Sure thing I will do the changes.

**Pull request merged**